DESCRIPTION

```diff
@@ -8,6 +8,11 @@ Authors@R:
              role = c("aut", "cre"), 
              email = "mikecp11@gmail.com", 
              comment = c(ORCID = "0000-0002-5288-8350")),
+      person(given = "Jim", 
```

Comment:
Please add same name as on paper and leave role as ctb

Reply:
- [x] Done